HWMgmt/OCPBaselineRedfishService.json

```diff
@@ -0,0 +1,571 @@
+                        "ServiceEnabled": {},
+                        "ServiceAddresses": {}
+                    }
+                },
+                "OutboundConnections": {
+                    "ReadRequirement": "Recommended"
+                },
+                "SupportedAccountTypes": {}
+            }
+        },
+        "Certificate": {
+            "ReadRequirement": "Mandatory",
+            "URIs": "/redfish/v1/Managers/{ManagerId}/NetworkProtocol/HTTPS/Certificates/{CertificateId}",
+            "PropertyRequirements": {
+                "CertificateString": {}
+            }
+        },
+        "CertificateService": {
+            "ReadRequirement": "Mandatory",
+            "PropertyRequirements": {
+                "CertificateLocations": {}
+            },
+            "ActionRequirements": {
+                "GenerateCSR": {
+                    "ReadRequirement": "Mandatory",
+                    "ActionInfo": "Mandatory"
+                },
+                "ReplaceCertificate": {}
+            }
+        },
+        "EthernetInterface": {
+            "ReadRequirement": "Mandatory",
+            "URIs": [
+                "/redfish/v1/Managers/{ManagerId}/EthernetInterfaces/{EthernetInterfaceId}"
+            ],
+            "PropertyRequirements": {
+                "MACAddress": {},
+                "SpeedMbps": {
+                    "ReadRequirement": "Supported"
+                },
+                "InterfaceEnabled": {},
+                "LinkStatus": {},
+                "Status": {
+                    "PropertyRequirements": {
+                        "Health": {},
+                        "State": {}
+                    }
+                },
+                "DHCPv4": {
+                    "ReadRequirement": "Recommended",
+                    "WriteRequirement": "Recommended",
+                    "PropertyRequirements": {
+                        "DHCPEnabled": {
+                            "ReadRequirement": "IfImplemented"
+                        }
+                    }
+                },
+                "DHCPv6": {
+                    "ReadRequirement": "Recommended",
+                    "WriteRequirement": "Recommended",
+                    "PropertyRequirements": {
+                        "OperatingMode": {
+                            "ReadRequirement": "IfImplemented"
+                        }
+                    }
+                },
+                "HostName": {
+                    "ReadRequirement": "Supported"
+                },
+                "FQDN": {
+                    "ReadRequirement": "Supported"
+                },
+                "NameServers": {},
+                "IPv4Addresses": {
+                    "Address": {},
+                    "AddressOrigin": {},
+                    "Gateway": {},
+                    "SubnetMask": {}
+                },
+                "IPv4StaticAddresses": {
+                    "ReadRequirement": "Recommended"
```

Comment:
Should we add the subordinate properties and mark them with write requirements?

Reply:
Yes, as in, "if you're showing / supporting static addresses, then they need to be RW so they can be updated here."